card/lib/card/format/render.rb

```diff
@@ -132,8 +133,14 @@ def add_class options, klass
         options[:class] = [options[:class], klass].flatten.compact * ' '
       end
 
+      def id_counter
+        return @parent.id_counter if @parent
+        @id_counter ||= 0
+        @id_counter += 1
+      end
```

Comment:
these are instance methods, so I don't think they'll insure a unique id across the page.

we could make another module and extend format with that to make it a class method...